### PR TITLE
sql-38: Add Toastr Notofication

### PR DESCRIPTION
**Pull request by VladTkach**
ngx-toastr v15

### CODE REVIEW

File: frontend/angular.json [hunk 1]
```diff
@@ -46,7 +46,7 @@
                         "tsConfig": "tsconfig.app.json",
                         "inlineStyleLanguage": "sass",
                         "assets": ["src/favicon.ico", "src/assets"],
-                        "styles": ["src/styles/index.sass"],
+                        "styles": ["src/styles/index.sass", "node_modules/ngx-toastr/toastr.css"],
```
Comment (Anatolii-Ivanov) [inline frontend/angular.json]: Please, don't do it
No node_modules in project

**Review by Anatolii-Ivanov — COMMENTED** [review]
Can you create a service, which will have methods to show toastrs? So people won't need to go into documentations and discover how it's done

File: frontend/angular.json (hunk 1 above)
Comment (VladTkach) [reply frontend/angular.json]: so I need to import it form another place?

Comment (VladTkach) [reply frontend/angular.json]: oh, I think we need just add import to index.sass

**Review by serhiiyasenev — DISMISSED** [review]

**Comment by serhiiyasenev** [comment]
please, resolve conflict

**Review by Anatolii-Ivanov — APPROVED** [review]

**Review by serhiiyasenev — APPROVED** [review]

**Pull request merged**